app/src/main/java/com/sopt/now/Home/MyPageFragment.kt

```diff
@@ -5,23 +5,27 @@ import android.util.Log
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.Toast
 import androidx.fragment.app.Fragment
-import com.sopt.now.User.UserDataInput
+import com.sopt.now.api.ServicePool
 import com.sopt.now.databinding.FragmentMyPageBinding
+import com.sopt.now.dto.ResponseUserProfile
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
 
 
-class MyPageFragment: Fragment() {
-    //Nullable하고, OnDestroyView()에서 null로 해제해주는 객체
+class MyPageFragment : Fragment() {
     private var _binding: FragmentMyPageBinding? = null
-    //NonNullable하고, get()을 이용하여 값을 넣어주는 객체
-    //binding은 항상 _binding을 반환하기 때문에 null이 될 수 없다. val로 선언하기
     private val binding: FragmentMyPageBinding
         get() = requireNotNull(_binding)
 
+    private var userId: String? = null
+
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
-        savedInstanceState: Bundle?
+        savedInstanceState: Bundle?,
     ): View? {
         _binding = FragmentMyPageBinding.inflate(inflater, container, false)
         return binding.root
```

Comment:
이미 View 타입이 초기화 이후 binidng.root를 반환하기 때문에 non null 타입으로 지정해주세요!
"
```suggestion
    ): View {
        _binding = FragmentMyPageBinding.inflate(inflater, container, false)
        return binding.root
```